test/micro/org/openjdk/bench/java/lang/foreign/pointers/Point.java

```diff
@@ -0,0 +1,40 @@
+package org.openjdk.bench.java.lang.foreign.pointers;
+
+import java.lang.foreign.GroupLayout;
+import java.lang.foreign.MemoryLayout;
+import java.lang.foreign.MemorySegment;
+import java.lang.foreign.ValueLayout;
+
+public class Point extends Struct<Point> {
+
+    Point(Pointer<Point> ptr) {
+        super(ptr);
+    }
+
+    int x() {
+        return ptr.segment.getAtIndex(NativeType.C_INT.layout(),  0);
+    }
+    int y() {
+        return ptr.segment.getAtIndex(NativeType.C_INT.layout(),  4);
```

Comment:
Shouldn't this just be `get`? (otherwise this seems like it would access at offset = 4 * 4)